plugins/threatminer/icon_threatminer/connection/connection.py

```diff
@@ -0,0 +1,18 @@
+import insightconnect_plugin_runtime
+from .schema import ConnectionSchema
+
+# Custom imports below
+from icon_threatminer.util.api import ThreatminerAPI
+
+
+class Connection(insightconnect_plugin_runtime.Connection):
+    def __init__(self) -> None:
+        super(self.__class__, self).__init__(input=ConnectionSchema())
+        self.api_client = None
+
+    def connect(self, params={}):
+        # pylint: disable=unused-argument
+        self.api_client = ThreatminerAPI()
+
+    def test(self):
+        ...
```

Comment:
Is there any basic get requests we could use to implement a connection test?

Reply:
Oh, yea. Sure 😄 Just missed it here